test/unit/eigensolver/test_reduction_to_band.cpp

```diff
@@ -308,16 +308,20 @@ auto checkResult(const SizeType k, const SizeType band_size, Matrix<const T, Dev
 
 template <class T, Backend B, Device D>
 void testReductionToBandLocal(const LocalElementSize size, const TileElementSize block_size,
-                              const SizeType band_size) {
+                              const SizeType band_size, const bool banded = false) {
```

Comment:
Is `banded = block_size == band_size` or something in that direction? Whatever it is, would it be possible to add the newly added test cases to the list of other test cases and infer `banded` from the test configuration?

Reply:
Ah, I see... yes, what you're suggesting sounds clearer at least to me! If you don't mind going for the enum I'd prefer that.